javax-money/src/test/java/com/fasterxml/jackson/datatype/money/CurrencyUnitDeserializerTest.java

```diff
@@ -0,0 +1,43 @@
+package com.fasterxml.jackson.datatype.money;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.jsontype.BasicPolymorphicTypeValidator;
+import org.javamoney.moneta.CurrencyUnitBuilder;
+import org.junit.Test;
+
+import javax.money.CurrencyUnit;
+import javax.money.UnknownCurrencyException;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertThrows;
+
+public final class CurrencyUnitDeserializerTest {
+
+    private final ObjectMapper unit = new ObjectMapper().findAndRegisterModules();
```

Comment:
This is wrong: should specifically add just Money module and not whatever might happen to be in classpath. Ideally would use helper method in test base class or utility/helper class